serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc

```diff
@@ -0,0 +1,21 @@
+= Dashboards in Kogito Serverless Workflow Tools extension
```

Comment:
How do you access this page ? Missing into the `nav.adoc` file ?

Reply:
ok